modules/social_features/social_activity/src/SocialActivityLazyBuilder.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Drupal\social_activity;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\views\ViewExecutableFactory;
+
+/**
+ * Class SocialActivityLazyBuilder.
+ *
+ * @package Drupal\social_activity
+ */
+class SocialActivityLazyBuilder {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The views executable factory.
+   *
+   * @var \Drupal\views\ViewExecutableFactory
+   */
+  protected $viewExecutable;
+
+  /**
+   * SocialActivityLazyBuilder constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   * @param \Drupal\views\ViewExecutableFactory $view_executable
+   *   The views executable factory.
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, ViewExecutableFactory $view_executable) {
+    $this->entityTypeManager = $entity_type_manager;
+    $this->viewExecutable = $view_executable;
+  }
+
+  /**
+   * Returns views render for lazy builder.
+   *
+   * @param string $view_id
+   *   The views ID.
+   * @param string $display_id
+   *   The views display ID.
+   * @param string $node_type
+   *   Node bundle.
+   * @param int $item_per_page
+   *   Items to display.
+   *
+   * @return array|null
+   *   Render array.
+   */
+  public function viewsLazyBuild($view_id, $display_id, $node_type, $item_per_page) {
+    // Get view.
+    $view_entity = $this->entityTypeManager
```

Comment:
I think similar things can be write in one line
$view_entity = $this->entityTypeManager->getStorage('view')->load($view_id);